.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,39 @@
-<!-- The text between the arrows are comments - they will not be visible on your PR. -->
-
-# About the PR
-<!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
+# Description
+<!--
+Describe the Pull Request here.
+What does it change?
+What other things could this impact?
+-->
 
+# TODO
+<!--
+If this PR is a work in progress, you can keep track of what you need to do here.
+A checklist can be make using the following syntax:
+- [ ] Task
+- [x] Completed Task
+- [ ] Other Task
+-->
 
 # Media
 <!--
-If applicable, add screenshots or videos to showcase your PR. Small fixes/refactors are exempt, but all PRs which make ingame changes
-(adding clothing, items, new features, etc) must include ingame media or the PR will not be merged.
-This makes it much easier for us to merge PRs and find media for progress reports. If you include media in your pull request, we
-may potentially use it in the progress reports, with clear credit given.
-
-Use screenshot software like Window's built in snipping tool, ShareX, Lightshot, or recording software like ShareX (gif), ScreenToGif, or Open Broadcaster Software (cross platform).
```

Comment:
I 100% understand your perspective for removing this line, but you need to keep in mind your position and where you're coming from.

This is, and I promise you I'm being honest, _not_ as obvious as you think it is to anyone who might be making a PR.